docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -0,0 +1,189 @@
+
+## Add a wallet address to an inbox
+
+:::code-group
+
+```jsx [React Native]
+await client.addAccount(walletToAdd)
+```
+
+```kotlin [Kotlin]
+client.addAccount(walletToAdd)
+```
+
+```swift [Swift]
+try await client.addAccount(newAccount: walletToAdd)
+```
+
+:::
+
+## Remove a wallet address from an inbox
+
+:::tip[Note]
+You cannot remove a recovery wallet.
+:::
+
+:::code-group
+
+```jsx [React Native]
+await client.removeAccount(recoveryWallet, addressToRemove)
+```
+
+```kotlin [Kotlin]
+client.removeAccount(recoveryWallet, addressToRemove)
+```
+
+```swift [Swift]
+try await client.removeAccount(recoveryAccount: recoveryWallet, addressToRemove: addressToRemove)
+```
+
+:::
+
+## Revoke all other installations
+
+Your user may encounter a case where a wallet address for one of their installation IDs gets compromised:
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/compromised-address.png" width="350px" />
+</div>
+
+In this case, you can revoke the address’ access to all other installations:
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/revoke-address.png" width="350px" />
+</div>
+
+:::code-group
+
+```jsx [React Native]
+await client.revokeAllOtherInstallations(recoveryWallet)
+```
+
+```kotlin [Kotlin]
+client.revokeAllOtherInstallations(recoveryWallet)
+```
+
+```swift [Swift]
+try await client.revokeAllOtherInstallations(signingKey: recoveryWallet)
+```
+
+:::
+
+## View the inbox state
+
+Find an `inboxId` for an address:
+
+:::code-group
+
+```jsx [React Native]
+const inboxId = await client.inboxIdFromAddress(address)
+```
+
+```kotlin [Kotlin]
+val inboxId = client.inboxIdFromAddress(address)
+```
+
+```swift [Swift]
+let inboxId = try await client.inboxIdFromAddress(address: address)
+```
+
+:::
+
+View the state of any inbox to see the addresses, installations, and other information associated with the `inboxId`.
+
+**Request**
+
+:::code-group
+
+```jsx [React Native]
+const state = await client.inboxState(true)
+const states = await client.inboxStates(true, [inboxId, inboxId])
+```
+
+```kotlin [Kotlin]
+val state = client.inboxState(true) 
+val states = client.inboxStatesForInboxIds(true, listOf(inboxID, inboxID))
+```
+
+```swift [Swift]
+let state = try await client.inboxState(refreshFromNetwork: true)
+let states = try await client.inboxStatesForInboxIds(
+	refreshFromNetwork: true,
+	inboxIds: [inboxID, inboxID]
+)
+```
+:::
+
+**Response**
+
+```json
+InboxState
+{
+	recoveryAddress: string,
+	addresses: string[],
+	installations: string[],
+	inboxId: string,
+}
+```
+
+## FAQ
+
+### What happens when a user removes a wallet address?
+
+- If a user has only one wallet address associated with their inbox:
+- If a user has multiple wallet addresses associated with their inbox:
+
+### How is the recovery address used?
+
```

Comment:
I need help with the anwers here...

Reply:
The recovery address is used to remove and revoke installations or addresses. You need the signer associated with the recovery address to sign the transactions for removing addresses or revoking installations. For example I can add a random person to my inboxId so they can see my groups and conversations and respond to them for me. However if I decide I no longer want them a part of my inbox I can remove. However that person cannot remove me from my own inbox because they do not have access to the recovery address signer.